Library/Formula/cmake.rb

```diff
@@ -15,6 +15,7 @@ class Cmake < Formula
   option "without-docs", "Don't build man pages"
 
   depends_on :python => :build if OS.mac? && MacOS.version <= :snow_leopard && build.with?("docs")
+  depends_on "curl"
```

Comment:
curl is provided by default on Mac OS, so this dependency should not apply to Mac OS. Linuxbrew aims to maintain compatibility with Homebrew and not change the behaviour of Homebrew when Linuxbrew is run on a Mac. This line should be
```ruby
depends_on "curl" unless OS.mac?
```